### PR TITLE
Support loading from local files and S3 with %seed, load samples from remote source

### PR DESCRIPTION
**Pull request by michaelnchin**
Issue #, if available: #44

Description of changes:
- Added new options to `%seed` to support loading from a specified file location.
    - `--language`: Specifies which language endpoint(`gremlin`,`cypher`, or `sparql`) to submit the queries to.
    - `--source`: Specifies the full path to a data file, or a directory containing multiple files, that you would like to load queries from. This option will also accept an S3 URI for any bucket folder or object that your local AWS credential grant permission to access.
    - `-f`/`--full-file-query`: If using Gremlin or openCypher, specifying this flag will submit the entire content of the file as a single query, instead of one query per line.
- Removed sample seed data from package. Samples will now be loaded from an external S3 bucket.

Example run with arguments to load from a local file:

`%seed --language Gremlin --source /Users/foo/seed_directory/data_file_0.txt -f --run`

Running `%seed` by itself will allow you to select the same options.

Screen Shot 2022-03-30 at 4 12 18 PM

Indicating an S3 URI as the source will generate a slightly different interface.

Screen Shot 2022-04-12 at 5 50 07 PM

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by krlawrence — CHANGES_REQUESTED** [review]
Given we now allow a user to tell us what type of data a file is, we may want to add tests for cases like the user says it is Gremlin, but the file contains Cypher (or even total garbage).

**Comment by michaelnchin** [comment]
Thanks for raising this point, will remove the hard-coded checks. Parser exceptions resulting from the request will be caught conventionally and displayed to the user. Can change this behavior later depending on user feedback.

File: src/graph_notebook/seed/queries/propertygraph/__init__.py [hunk 1]
```diff
@@ -1,4 +0,0 @@
-"""
-Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (bechbd) [inline src/graph_notebook/seed/queries/propertygraph/__init__.py:2]: Is there a reason we are removing this copyright?

**Review by bechbd — COMMENTED** [review]
It looks like we are removing all the seed command files but I don't see them added back anywhere?  How are you thinking this will work?
Does this need to go through a security review as we are now including reading from S3?

File: src/graph_notebook/seed/queries/propertygraph/__init__.py (hunk 1 above)
Comment (michaelnchin) [reply src/graph_notebook/seed/queries/propertygraph/__init__.py:2]: File was removed as part of the `.../seed/queries` directory. The queries folder and its contents have been relocated to our public production S3 bucket at `s3://aws-neptune-notebook/queries/`.

**Comment by michaelnchin** [comment]
The seed command files at `.../seed/queries` have been relocated to our public production S3 bucket at `s3://aws-neptune-notebook/queries/`. 

`.../seed/load_query.py` has been changed accordingly: https://github.com/aws/graph-notebook/pull/247/files#diff-9c7db7df655d9c8b64dfc3335a48869e6f10362bb760ab9c6e4a56d86d5fbbe1

We'll now use Boto3 to create an S3 request to download the specified dataset from this bucket into the local Jupyter notebook directory, then load the data as normal.

**Comment by michaelnchin** [comment]
Closing this PR to divide up the local and S3 changes.

**Pull request closed without merging**